api/src/main/java/org/openmrs/module/providermanagement/ProviderManagementUtils.java

```diff
@@ -99,6 +104,66 @@ public static Date clearTimeComponent(Date date) {
 
         return cal.getTime();
     }
+
+    public static List<ProviderPersonRelationship> getSupervisors(Provider provider)
```

Comment:
Thanks @mogoodrich ! I documented the new functions and I am going to push it along with my next PR which should be soon.